index.ts

```diff
@@ -32,7 +33,18 @@ for (const fp of filePaths) {
     const key = path.parse(fp).name;
 
     const src = fs.readFileSync(fp, { encoding: 'utf-8'});
-    const data = YAML.parse(src);
+    const data = YAML.parse(src) as FeatureData;
+
+    // Compute Baseline high date from low date.
+    if (data.status?.baseline_high_date) {
+        throw new Error(`baseline_high_date is computed and should not be used in source YAML. Remove it from ${key}.yml.`);
+    }
+    if (data.status?.baseline === 'high') {
+        const lowDate = Temporal.PlainDate.from(data.status.baseline_low_date);
+        const highDate = lowDate.add({ months: 30 });
```

Comment:
Should we put `30` in a const near the top of the file to avoid it being a magic number here?

`const NUMBER_OF_MONTHS_FROM_LOW_TO_HIGH = 30;`

Reply:
Good idea, I added a `monthsFromBaselineLowToHigh` (matching the style of the other constant up there)